scripts/presubmit.sh

```diff
@@ -97,6 +97,11 @@ main() {
     go build ${goflags} ./...
 
     echo 'running go test'
+    # Install test deps so that individual test runs below can reuse them.
+    echo 'getting test deps'
+    go get -t
```

Comment:
Without "./...", this will only do so for the "trillian" package.

Reply:
Done